views/drones/list.hbs

```diff
@@ -1 +1,32 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<div>
+
+  <h2 class="title">List of available drones</h2>
+
+  <hr>
+
+  <div class="drones">
+    {{#each DronesFromDB}}
+
+    <div class="dronebox">
+      
+      <h3>{{name}}</h3>
+      <div class="middledronebox">Number of propellers: {{propellers}}. </div>
+      <div class="speed-drone" >Max. speed: {{maxSpeed}} m per second.</div>
+      <form action="drones/{{_id}}/edit" method="GET">
+        <button type="submit">Edit drone</button>
+      </form>
+      <form class='deletebttn' action="drones/{{_id}}/delete" method="post">
+        <button type="submit">Delete drone</button>
+      </form>
+      
+    </div>
+
+    {{/each}}
+  </div>
+
+  <a class='createdrone' href="/drones/create"> Create a drone </a>
```

Comment:
I recommend using camelCase notation when you create a class made of two words `createDrone` (it's best practice)

Reply:
fixed the console.logs, the path, I changed the form for an anchor (I left i as a button for visual purposes, it was easier than modifying an anchor to look like another button) and I will be more careful with the names, I just pushed it to my forked repository. Thanks for the review! :)